docs/coding-conventions.md

```diff
@@ -0,0 +1,76 @@
+# Move Coding Conventions
+
+This section lays out some basic coding conventions for Move that the Move team has found helpful. These are only recommendations, and you should feel free to use other formatting guidelines and conventions if you have a preference for them.
```

Comment:
I think we should unify coding conventions in this repo.

Reply:
Yes, add this